.github/workflows/tests.yml

```diff
@@ -41,6 +41,16 @@ jobs:
         ports:
           - 6379:6379
 
+      redis-cluster:
+        image: redisfab/redis-py-cluster:6.2.6
```

Comment:
This seems to ridgit. We should support 4, 5, 6 and 7 no?

Reply:
Only 5,6,7 are supported. But cluster support in 6.x covers most cases - if anything we should use the redisenv clusters @vladvildanov

Reply:
@tillkruss @chayim Let's stick to other client practice, are you testing agains all cluster version or latest one?

Reply:
Against that spreadsheet; 7.x is fine for cluster